tests/unit/components/HdToggle.spec.js

```diff
@@ -59,6 +61,29 @@ describe('HdToggle', () => {
     expect(wrapper.emitted().toggle).toBeTruthy();
   });
 
+  it('renders actions menu when provided props', async () => {
+    const wrapper = wrapperBuilder();
+
+    wrapper.setProps({
+      actions: [
+        {
+          name: 'delete',
+          label: 'Delete',
+          icon: TEST_SVG,
+        },
+        {
+          name: 'edit',
+          label: 'Edit',
+          icon: TEST_SVG,
+        },
+      ],
+    });
+
+    await wrapper.vm.$nextTick();
+
+    expect(wrapper.html()).toMatchSnapshot();
```

Comment:
```suggestion
```
`HdToggle` already has a snapshot test, so it would be better to test only the actions behavior here:
- Check for the existence of the actions provided
- Check that when an action is clicked it emits an event with its name

Reply:
Got it, I have re-written the HdToggle test 👍